builder/imports/s2n.py

```diff
@@ -8,6 +8,7 @@
     'targets': ['linux', 'android'],
     'test_steps': [],
     'build_tests': False,
+    'cmake_args': ['-DUNSAFE_TREAT_WARNINGS_AS_ERRORS=OFF']
```

Comment:
Could you add a comment as to why we are doing this here?

Reply:
Sure